src/CommunityToolkit/Xamarin.CommunityToolkit/Effects/Window/PlatformWindowEffect.uwp.cs

```diff
@@ -39,12 +35,55 @@ protected override void OnElementPropertyChanged(PropertyChangedEventArgs args)
 			}
 		}
 
-		public void SetStatusBarColor(Color color)
+		public void SetStatusBarColor(Windows.UI.Color color)
 		{
+			if (ApiInformation.IsTypePresent(typeof(StatusBar).FullName ?? string.Empty))
+			{
+				var statusBar = StatusBar.GetForCurrentView();
+				if (statusBar != null)
+				{
+					statusBar.BackgroundColor = color;
+				}
+			}
+			else
+			{
+				var titleBar = ApplicationView.GetForCurrentView()?.TitleBar;
+				if (titleBar != null)
+				{
+					titleBar.BackgroundColor = color;
+				}
+			}
 		}
 
 		public void SetStatusBarStyle(StatusBarStyle style)
 		{
+			var foregroundColor = Color.Default;
+			switch (style)
+			{
+				case StatusBarStyle.LightContent:
+					foregroundColor = Color.White;
+					break;
+				case StatusBarStyle.DarkContent:
+					foregroundColor = Color.Black;
+					break;
+			}
```

Comment:
```suggestion
			foregroundColor = style switch
			{
		     	StatusBarStyle.LightContent => Color.White,
		    	StatusBarStyle.DarkContent => 	Color.Black;
			};
```
Could we use a more compact syntax?

Reply:
Good point! Didn't noticed it could be used there. Converted that switch to switch expression